cvat-sdk/cvat_sdk/masks.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import math
+from collections.abc import Sequence
+
+import numpy as np
+from numpy.typing import ArrayLike
+
+
+def encode_mask(bitmap: ArrayLike, /, bbox: Sequence[float]) -> list[float]:
```

Comment:
Why the bbox is required?

Reply:
Mainly because I thought I could get away with it. 🙂 Torchvision provides a bounding box with each mask for free, and so does Ultralytics, so I figured it was common enough that I didn't need to implement automatic bbox determination.

If you think it's warranted, I could do it, though.